addons/website_slides/models/slide_channel.py

```diff
@@ -282,6 +282,7 @@ def _get_default_enroll_msg(self):
     upload_group_ids = fields.Many2many(
         'res.groups', 'rel_upload_groups', 'channel_id', 'group_id', string='Upload Groups',
         help="Group of users allowed to publish contents on a documentation course.")
+    website_background_image_url = fields.Char('Background image URL', compute='_compute_website_background_image_url')
```

Comment:
(Replying here as I can't reply to other comment).

> I went for a compute for performance reasons when we are on pages with multiple records (a course page or the profile, in which cases the image_256 is enough. In all cases where we would need a larger image, it will be used with field and a widget specifying the size.
Please let me know if you agree with this (and I'll update the commit message accordingly).
Antother option would be indeed a _get_background_image_url which could allow for parameters such as the requested size.

That seems fine, however I don't understand the "size" parameter in the compute.
As far as I know, it's not possible to pass parameters to a compute method of a field (unless we call it manually, but that's not really a good practice).